code/processes/wdb.q

```diff
@@ -151,26 +151,37 @@ savetables:$[writedownmode~`partbyattr;savetablesbypart;savetables];
 
 savetodisk:{[] savetables[savedir;getpartition[];0b;] each tablelist[]};
 
-/- eod - flush remaining data to disk
 endofday:{[pt;processdata]
-	.lg.o[`eod;"end of day message received - ",spt:string pt];
+        .lg.o[`eod;"end of day message received - ",spt:string pt];
```

Comment:
can we keep whitespace-only changes to separate PRs please? makes it less clear where there is meaningful change